projects/cdk/date-time/date-fillers.ts

```diff
@@ -1,10 +1,18 @@
 import {inject, InjectionToken} from '@angular/core';
+import {TuiDateMode} from '@taiga-ui/cdk/types';
 import {RANGE_SEPARATOR_CHAR} from './date-time';
 
-// TODO: think about yyyy.mm.dd format
-export const TUI_DATE_FILLER = new InjectionToken<string>('date filler for Taiga UI', {
-    factory: () => `dd.mm.yyyy`,
-});
+/**
+ * TODO: think about yyyy.mm.dd format
+ * TODO: Remove return type string in 3.0
+ * @deprecated return type will be narrowed down to just TuiDateMode (without string) in next major version
+ */
+export const TUI_DATE_FILLER = new InjectionToken<TuiDateMode | string>(
+    'date filler for Taiga UI',
+    {
+        factory: () => `dd.mm.yyyy`,
+    },
+);
 
 export const TUI_DATE_RANGE_FILLER = new InjectionToken<string>(
```

Comment:
There are also `InputDateRange` and `InputDate` components. If we are switching to adding those strings to translations, we need to do it there as well.

Reply:
done